Snake/GameBoard.java

```diff
@@ -1,253 +1,286 @@
 import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
 import java.util.Random;
 
+import javax.imageio.ImageIO;
+import javax.swing.ImageIcon;
+
 /**
- * Represents the environment where the Snake moves a food spawns.
- * <br/>
- * There are some special rules as to how the Snake can move. If the Snake's size
- * is 1, it can move in any direction. If the Snake's size is greater than 1, it
- * cannot move 180 degrees. Example: if the Snake is moving right, it cannot
- * immediately change its direction to left because it would run into itself.
+ * Represents the environment where the Snake moves a food spawns. <br/>
+ * There are some special rules as to how the Snake can move. If the Snake's
+ * size is 1, it can move in any direction. If the Snake's size is greater than
+ * 1, it cannot move 180 degrees. Example: if the Snake is moving right, it
+ * cannot immediately change its direction to left because it would run into
+ * itself.
  */
-class GameBoard  {
```

Comment:
According to the PR you've modified this whole class. This was probably a result of reformatting the code.
Please revert your changes so that you only change the lines that you need to change.